### PR TITLE
feat: allow arm64 architecture mapping in ci deps subcommand

### PR DESCRIPTION
**Pull request by alnr**

### CODE REVIEW

**Comment by alnr** [comment]
With `make skaffold-dev`, a single change in backoffice now takes about 30s from file saved to new version receiving traffic in k3d.

**Comment by CaptainStandby** [comment]
lgtm

**Review by CaptainStandby — APPROVED** [review]
lgtm

**Review by CaptainStandby — APPROVED** [review]

File: cmd/dev/ci/deps/url_test.go [hunk 1]
```diff
@@ -55,13 +56,19 @@ func TestDefaultURL(t *testing.T) {
...
 	comp1 := Component{}
 	_ = comp1.getComponentFromConfig("test/customArchURL.yaml")
+
 	url, err := comp1.getRenderedURL("darwin", "amd64")
 	assert.Nil(t, err, "Expected no Error!")
-	assert.Equal(t, customArchURL, url)
+	assert.Equal(t, customArchURL1, url)
+
+	url, err = comp1.getRenderedURL("darwin", "arm64")
+	assert.Nil(t, err, "Expected no Error!")
```
Comment (aeneasr) [inline cmd/dev/ci/deps/url_test.go:70]: @alnr please use `assert.NoError` or `assert.IsError` or `assert.AsError` for checking for errors

**Pull request merged**